test/typed_buffers_test.dart

```diff
@@ -267,10 +266,17 @@ void testIntBuffer(
   expect(buffer.length, equals(samples.length));
 
   // Both values are in `samples`, but equality is performed without rounding.
-  expect(buffer.contains(min - 1), isFalse);
-  expect(buffer.contains(max + 1), isFalse);
+  // For signed 64 bit ints, min and max wrap around, min-1=max and max+1=min
+  if (bits == 64) {
+    // TODO(keertip): fix tests for Uint64 / In64 as now Uints are represented
+    // as signed ints.
+    expect(buffer.contains(min - 1), isTrue);
+  } else {
+    expect(buffer.contains(min - 1), isFalse);
```

Comment:
I see no reason these tests can't run on 64-bit values too. 
The min and max values might not really be min and max (they're 0 and -1 when represented as Int64), but the `Uint64List.contains` method will still do `int.==` to figure out if the argument is in the list, and `max + 1` is 0, and that's definitely in the list.

So, all in all, I don't see a problem with keeping the test as-is (but maybe commenting that `Uint64List` acts exactly like `Int64List` because the `int` type cannot distinguish them.

The `Uint64List` tests will not work on JavaScript anyway because `Uint64List` doesn't exist.

Reply:
For Uint64 
`min is 0, min-1 = -1`
`max is -1, max+1 = 0`

and for Int64
`min is -9223372036854775808, min-1 = 9223372036854775807`
`max is 9223372036854775807, max+1 = -9223372036854775808`

so these lines
```
  expect(buffer.contains(min - 1), isFalse);
  expect(buffer.contains(max + 1), isFalse);
```
fail. Since it wraps around, the values are contained in the buffer.

Reply:
For Uint64 
`min is 0, min-1 = -1`
`max is -1, max+1 = 0`

and for Int64
`min is -9223372036854775808, min-1 = 9223372036854775807`
`max is 9223372036854775807, max+1 = -9223372036854775808`

so these lines
```
  expect(buffer.contains(min - 1), isFalse);
  expect(buffer.contains(max + 1), isFalse);
```
fail. Since it wraps around, the values are contained in the buffer.

Reply:
Ah, so `contains` didn't restrict the argument number to 64 bits before, but now it does. Makes sense. In that case, carry on!